### PR TITLE
RS+RT/YJ/RULE2-33

### PR DESCRIPTION
**Pull request by yunjoonjung-PNNL**
```get_primary_secondary_loops``` hasn't been integrated into this rule (**TODO** comments were left where ```get_primary_secondary_loops``` function is needed).

I will check this out again once ```get_primary_secondary_loops``` function is merged. Thank you!

### CODE REVIEW

File: rct229/rules/section22/section22rule33.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        rmi_b = context.baseline
+        baseline_system_types_dict = get_baseline_system_types(rmi_b)
+        # create a list contains all HVAC systems that are modeled in the rmi_b
+        available_type_lists = [
+            hvac_type
+            for hvac_type in baseline_system_types_dict.keys()
+            if len(baseline_system_types_dict[hvac_type]) > 0
+        ]
+        # primary_secondary_loop_dictionary = get_primary_secondary_loops(rmi_b) # TODO this should be updated!!
+        primary_secondary_loop_dictionary = {
```
Comment (weilixu) [inline rct229/rules/section22/section22rule33.py:41]: Need to replace this with the actual function now

**Review by weilixu — APPROVED** [review]
Approved with one comment needs to be addressed

**Review by charliepnnl — APPROVED** [review]
Approved with two tiny style-type comments

File: rct229/rules/section22/section22rule33.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+
+class Section22Rule33(RuleDefinitionBase):
+    """Rule 33 of ASHRAE 90.1-2019 Appendix G Section 22 (Chilled water loop)"""
+
+    def __init__(self):
+        super(Section22Rule33, self).__init__(
+            rmrs_used=UserBaselineProposedVals(False, True, False),
+            id="22-33",
+            description="Baseline chilled water system that does not use purchased chilled water must only have no more than one CHW plant.",
```
[the remark this answers is not in the document]
Comment (yunjoonjung-PNNL) [reply rct229/rules/section22/section22rule33.py:30]: I'll follow the discussion results with Weili for this comment.

File: rct229/rules/section22/section22rule33.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+            rmr_context="ruleset_model_instances/0",
+        )
+
+    def is_applicable(self, context, data=None):
+        rmi_b = context.baseline
+        baseline_system_types_dict = get_baseline_system_types(rmi_b)
+        # create a list contains all HVAC systems that are modeled in the rmi_b
+        available_type_lists = [
```
[the remark this answers is not in the document]
Comment (yunjoonjung-PNNL) [reply rct229/rules/section22/section22rule33.py:38]: Fixed.

**Comment by yunjoonjung-PNNL** [comment]
This branch will be merged after discussing whether the ```only``` in included or not in the description.

File: rct229/rules/section22/section22rule33.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (weilixu) [reply rct229/rules/section22/section22rule33.py:30]: Per discussion, remove the only (approved by RDS team)

**Review by weilixu — APPROVED** [review]

**Pull request merged**